x-pack/plugins/security_solution/public/resolver/view/panels/related_event_detail.tsx

```diff
@@ -90,6 +50,39 @@ const TitleHr = memo(() => {
 });
 TitleHr.displayName = 'TitleHR';
 
+// Indicates if the User Agent supports <wbr/>
+const noWBRTagSupport = document.createElement('wbr') instanceof HTMLUnknownElement;
```

Comment:
don't think this is needed, the tag is supported in IE8+

Reply:
going off caniuse:
![image](https://user-images.githubusercontent.com/1087008/91314399-8453b880-e784-11ea-97af-a28d5793a44e.png)

Reply:
We don't support IE 11

Reply:
removed it: https://github.com/elastic/kibana/pull/75926/commits/4c48fc5da21f6bc09773a3eef16f436d332e7e9f